bundles/org.openhab.binding.senechome/src/main/java/org/openhab/binding/senechome/internal/SenecHomeHandler.java

```diff
@@ -118,6 +118,13 @@ private void refresh() {
         refreshCache.getValue();
     }
 
+    private void updateChannelState(String channelConstant, org.openhab.core.types.State channelState) {
```

Comment:
`State` should be added as import:
```suggestion
    private void updateChannelState(String channelConstant, State channelState) {
```